PublicMapping/districtbuilder
# Update release process as part of Github actions switchover

* Change name for rollbar from Jenkins to Github_Actions
 * Streamline release workflow, remove manual staging test - and make sure
   we check to see if staging built OK.
 * Remove redundant slack notifications.  Rollbar already provides one.
 * Remove Release variant of Jenkinsfile

### Checklist

These are minor changes related to #1245 and #1255

## Testing Instructions

- Confirm rollbar stops saying Jenkins 
- Confirm new release process makes sense
